queries.sql

```diff
@@ -7,3 +7,58 @@ SELECT name,escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3 ;
+/********transactions***********/
+ BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+UPDATE 11
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+COMMIT;
+BEGIN
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon' ;
+UPDATE 6
+SELECT * FROM animals;
+UPDATE animals
+SET species = 'pokemon'
+WHERE species = '';
+UPDATE 5
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
+BEGIN;
+DELETE
+FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+ SELECT * FROM animals;
+BEGIN;
+DELETE 
+FROM animals
+WHERE date_of_birth > '2022-01-01';
+SAVEPOINT afterdelete;
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+ROLLBACK TO SAVEPOINT afterdelete;
+ROLLBACK
+SELECT * FROM animals;
+UPDATE animals
+SET weight_kg =  weight_kg * -1
+WHERE weight_kg < 0 ;
+UPDATE 4
+SELECT * FROM animals;
+COMMIT;
+/************Aggregation*****************/
+SELECT COUNT(*)  FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG ( weight_kg) FROM animals;
+SELECT MAX(escape_attempts)
+SELECT species, MIN(weight_kg),  MAX (weight_kg)
```

Comment:
- [ ] Great job working here but i think it would be great if you also complete the query here `GROUP BY species;` I could see you add it in the screenshot and it is right but please add it here as well 💯✨✅